pyvex/lift/libvex.py

```diff
@@ -3,49 +3,110 @@
 l = logging.getLogger('pyvex.lift.libvex')
 l.setLevel(20) # Shut up
 
-from . import Lifter, register
+from .. import stmt
+from . import Lifter, register, LiftingException
 from .. import pvc, ffi
+from ..enums import default_vex_archinfo, vex_endness_from_string
 
 _libvex_lock = threading.Lock()
 
 SUPPORTED = {'X86', 'AMD64', 'MIPS32', 'MIPS64', 'ARM', 'ARMEL', 'ARMHF', 'AARCH64', 'PPC32', 'PPC64'}
 
+VEX_MAX_INSTRUCTIONS = 99
+VEX_MAX_BYTES = 400
```

Comment:
this 400 value is actually a lie, it's a lot bigger! at least 1000, I can't recall

Reply:
libvex.h says 5000